fgpyo/util/metric.py

```diff
@@ -192,7 +192,12 @@ def _parsers(cls) -> Dict[type, Callable[[str], Any]]:
         return {}
 
     @classmethod
-    def read(cls, path: Path, ignore_extra_fields: bool = True) -> Iterator["Metric[MetricType]"]:
+    def read(
+        cls,
+        path: Path,
+        ignore_extra_fields: bool = True,
+        comment_prefix: str = "#",
```

Comment:
Is this backwards incompatible?  Should we bump the minor version?

Reply:
Good question, it isn't.

One possible solution for backwards compatibility - I could update `Metric._read_header` to take an `Optional` comment prefix, and return the first line if the prefix is `None` (i.e. no skipping of prefixed _or_ blank lines)

Is that something you'd like to see, or should I leave as is and bump the version?

Reply:
I might suggest `comment_symbol`?

Also i agree with @nh13 that ideally the method gets expanded to include both:

```python
comment_symbol: Optional[str] = None,
skip_blank_lines: bool = False
```

I might even argue for `skip_blank_lines` to even be a regex that defines lines to skip ... so e.g. I could use `skip_lines_matching = "$\s*^"` to skip stupid lines that are all tabs/spaces but no content!